plugins/videobasedtracker/UVCImageSource.cpp

```diff
@@ -84,13 +84,25 @@ namespace vbtracker {
         //@}
 
       private:
-        uvc_context_t *uvcContext_;
-        uvc_device_t *camera_;
-        uvc_device_handle_t *cameraHandle_;
+        typedef std::unique_ptr<uvc_frame_t, decltype(&uvc_free_frame)>
```

Comment:
OK, have already fixed this and just pushed! Hopefully you don't mind the slight bit of template programming... This should be a very common pattern, so I wouldn't be surprised if something already exists?